explainer.md

```diff
@@ -54,11 +54,50 @@ Date: 2019-07-17
 
 ## Overview
 
-The **Badging API** is a proposed Web Platform API allowing websites to apply badges to sets of pages on their origin. These badges can be displayed in a location related to their documents, such as tab favicons.
+The **Badging API** is a proposed Web Platform API allowing websites to apply
+badges (small status indicators) to pages or sets of pages on their origin. We
+are deliberately agnostic about which contexts a badge can appear in, but have
+two fairly different contexts in mind:
+
+* "Document" contexts, associated with an open document, such as when a badge is
+  shown on or near the page's icon in a browser tab.
+* "Handle" contexts, associated with a handle or link to a site somewhere in the
+  user agent or operating system UI, but not necessarily associated with a
+  running document, such as a bookmark or [installed web
+  app](https://www.w3.org/TR/appmanifest/#installable-web-applications) icon.
+
+Here is a mock of a badge being applied to the page tab:
+
+![Mock of a badge being applied to the page tab](images/tab-badge.png)
+
+This use case is already satisfied today with sites dynamically setting either
+their favicon or title to include some status indicator. The use of an explicit
+badge API has a number of advantages over the "hack" ways:
+
+* The badge can appear outside (completely, or partially overlapping) of the
+  favicon, leaving more room for the site's brand image.
+* The badge is meaningful to the user agent, which can present it to the user in
+  various ways, such as announcing it verbally to a user using a screen reader.
+* Badges can be displayed with a consistent style across websites, chosen by the
+  user agent.
+* User agents can provide a way for users to disable badges on a per-site or
+  global basis.
+
+The "handle" context is more nebulous because it means associating a badge with
+any place the user agent shows an icon representing a site, page or app. For
+example, it could be applied to an icon in the bookmark bar, or a "commonly
+visited sites" index on the new tab page.
+
+![Mock of a badge being applied to the bookmark bar](images/bookmark-badge.png)
```

Comment:
The image is really small, so you don't really get any context of what the picture is of. Maybe use a more zoomed out image?

Reply:
Done.